Sources/SwiftIfConfig/IfConfigEvaluation.swift

```diff
@@ -0,0 +1,516 @@
+      // is an underscore followed by an integer.
+      guard let argExpr = call.argumentList.singleUnlabeledExpression,
+        let arg = argExpr.simpleIdentifierExpr,
+        let argFirst = arg.first,
+        argFirst == "_",
+        let expectedPointerBitWidth = Int(arg.dropFirst())
+      else {
+        result = nil
+        break
+      }
+
+      // If the build configuration doesn't know the pointer bit width, fail.
+      guard let targetPointerBitWidth = configuration.targetPointerBitWidth else {
+        throw IfConfigError.unhandledFunction(name: fnName, syntax: ExprSyntax(call))
+      }
+
+      result = targetPointerBitWidth == expectedPointerBitWidth
+
+    case .swift:
+      result = try doVersionComparisonCheck(configuration.languageVersion)
+
+    case .compiler:
+      result = try doVersionComparisonCheck(configuration.compilerVersion)
+
+    case ._compiler_version:
+      // Argument is a single unlabeled argument containing a string
+      // literal.
+      guard let argExpr = call.argumentList.singleUnlabeledExpression,
+        let stringLiteral = argExpr.as(StringLiteralExprSyntax.self),
+        stringLiteral.segments.count == 1,
+        let segment = stringLiteral.segments.first,
+        case .stringSegment(let stringSegment) = segment
+      else {
+        // FIXME: better diagnostic here
+        throw IfConfigError.unknownExpression(condition)
+      }
+
+      let versionString = stringSegment.content.text
+      let expectedVersion = try VersionTuple(parsingCompilerBuildVersion: versionString, argExpr)
+
+      guard let actualVersion = configuration.compilerVersion else {
+        throw IfConfigError.unhandledFunction(name: fnName, syntax: argExpr)
+      }
+
+      return actualVersion >= expectedVersion
+
+    case .canImport:
+      // Retrieve the first argument, which must not have a label. This is
+      // the module import path.
+      guard let firstArg = call.argumentList.first,
+        firstArg.label == nil
+      else {
+        throw IfConfigError.canImportMissingModule(syntax: ExprSyntax(call))
+      }
+
+      let importPath = firstArg.expression.trimmedDescription.split(separator: ".")
+      // FIXME: Check to make sure we have all identifiers here.
+
+      // If there is a second argument, it shall have the label _version or
+      // _underlyingVersion.
+      let version: CanImportVersion
+      if let secondArg = call.argumentList.dropFirst().first {
+        if secondArg.label?.text != "_version" && secondArg.label?.text != "_underlyingVersion" {
+          throw IfConfigError.canImportLabel(syntax: secondArg.expression)
+        }
+
+        let versionText: String
+        if let stringLiteral = secondArg.expression.as(StringLiteralExprSyntax.self),
+          stringLiteral.segments.count == 1,
+          let firstSegment = stringLiteral.segments.first,
+          case .stringSegment(let stringSegment) = firstSegment
+        {
```

Comment:
Yes, the C++ version of this code (`getCanImportVersion`) allows string literals for the version. Per my other comment, I don't understand why we're parsing `canImport` so completely in the parser, when it can be a normal `CallExprSyntax`.